config/schemas/config.yaml

```diff
@@ -4061,6 +4061,215 @@ properties:
           customTemplate:
             title: Slack Custom Template
             type: string
+      runbookUrls:
+        title: Alert runbooks
+        description: |-
+          Configure runbooks for alerts
+
+          Runbooks can be configured on an alert group level or per individual alert
+        type: object
+        additionalProperties: false
+        properties:
+          alertmanager:
+            title: Runbooks for alertmanager alerts
+            description: |-
+              Runbooks for alertmanager alerts
+
+              Example:
+
+              group: https://runbooks.prometheus-operator.dev/runbooks/alertmanager/
+              AlertmanagerFailedReload: https://runbooks.prometheus-operator.dev/runbooks/alertmanager/alertmanagerfailedreload/
+              AlertName: link-to-specific-alert-runbook
+
+              Uses upstream runbooks by default
+
+              https://runbooks.prometheus-operator.dev/runbooks/
```

Comment:
I think that this description is nice, but I see that this one is the only one with the examples. It would be nice if we had the examples for all groups, not just alertmanager, but I also don't want us to duplicate this text for each.

Suggestion: Could you add this to a template instead and add a reference here? Then we can get it to all without having to duplicate the text. You could have two templates, one with the text about using upstream runbook by default and one without. What do you think about this? Did you already have some similar plan?

Reply:
Are all these objects different? I wonder if there's any way to have reusable common bits?

Reply:
Hm, but for validation? I guess `"additionalProperties":{"type":"string"}` might be good enough

Reply:
:+1: I think this works, at least it validates

https://github.com/elastisys/compliantkubernetes-apps/pull/2438/commits/c835a3c118814af2f1f2df44b57a3b91e17ffa9b